### ISSUE TITLE
UXR: Website Redesign: "Donate" Page 2- UI

### ISSUE THREAD

**Issue by hgordon6634** [orcasound/orcahome#52]
This is a second version of this page that needs to be deployed. 

***User Story***
**AS** a New Orcahome user **I WANT** to see an easy to read webpage when I navigate to the Orcasound "Donate" page **SO THAT** I can easily donate to Orcasound's efforts.

***Description***
The intent of this story is to implement the UI finalized design from the Orcasound UX/UI team for the website redesign of the "Learn" Page. This Issue covers the Donation Wizard portion of the page (see image below). 

NOTES: 
-This is the UI only, although links to the partner organization's website can be included in the buttons if possible. 
-Header/Footer already completed on different pull request #14 
-Use Material UI for the styling 
 
***Acceptance Criteria***
**GIVEN** I open the Orcasound Donate page **WHEN** I look at the page **THEN** I see a UI that matches the attached UX/UI design.

https://www.figma.com/file/YdvbAjHETokV2x1LBHaElj?node-id=1015:1110#410004342
![image](https://user-images.githubusercontent.com/93286885/152079375-211ece13-77eb-4bb3-8f87-e4bfae65399d.png)

**Comment by ishubham21** [comment]
Hi @hgordon6634 can I work on this issue?

**Comment by MohitSharma-21** [comment]
@hgordon6634 Now I would like to work on this issue .

**Comment by hgordon6634** [comment]
Hey @MohitSharma-21 @ishubham21 ! Actually ShwetKhatri2001 is currently working on this page in a different story (issue #51 ), so his needs to be completed first before this issue can be worked on (which is why it's tagged "blocked"). You are welcome to check on his progress and ask to pick this one up after he's done.

**Comment by UXBrendan** [comment]
@ShwetKhatri2001 - If you are still working on issue [51](https://github.com/orcasound/orcahome/issues/51), please mark as complete so work on this issue can be started.

**Comment by UXBrendan** [comment]
@ShwetKhatri2001 - Yes, I see that now. I apologize for my newbie mistake! Thanks!

**Comment by UXBrendan** [comment]
@ishubham21 and @MohitSharma-21 

Let me know the status of this issue. Have you completed it, or can it move toward completion?

**Comment by UXBrendan** [comment]
@ishubham21 

Just checking in with you- how is dev work going for this issue? Anything you need from me?

**Comment by UXBrendan** [comment]
@ishubham21 

Just checking in with you- how is dev work going for this issue? Anything you need from me?

**Comment by UXBrendan** [comment]
@ishubham21 

Have you included linking all hypertext and button links to appropriate target pages for "'Learn' Page 2- Top- Donation Wizard"? If not, I'll create a new issue.

**Comment by UXBrendan** [comment]
This project is put on hold until UX Design work is complete.

**Comment by UXBrendan** [comment]
UX user story before this issue can be sent to production: https://github.com/orcasound/product/issues/34

**Comment by UXBrendan** [comment]
@UXBrendan 

Rewrite user story to include new design work for the donate page done by @CourtneyDruzak

**Comment by UXBrendan** [comment]
Could we A/B test this Donate page with the version that was deployed?

**Comment by BT-Orcasound-Product** [comment]
**Standup update — 2026-05-07**

Sprint scope decision: this issue is **back IN scope** for the current sprint. Originally deferred at kickoff (5/04) as needing discussion / blocked, but during today's 1:1 standup with @fdsafewa we agreed she will pick this up after #169 (Call Catalog) ships. Removing the `status: blocked` and `status: needs discussion` labels.

Sequence: #169 → #171 → #210 → #216 → #52.

Note: this won't make the 5/15 deployment cut — plan for late-sprint or post-sprint deploy.

**Comment by BT-Orcasound-Product** [comment]
**Asset reference (added 2026-05-18, Sprint 2 prep).**

The Donate page partner list is now externalized as canonical data in the repo. Implementation of this issue's second Donate page should source partner cards from the same data file, not re-implement the list.

- **Canonical partner data:** [`src/data/donatePartners.json`](https://github.com/orcasound/orcahome/blob/main/src/data/donatePartners.json). Eleven 501(c)(3) network partners. Fields: `icon`, `name`, `description`, `linkTo`.
- **Reference display component:** [`src/components/Donate/DonatePartners.jsx`](https://github.com/orcasound/orcahome/blob/main/src/components/Donate/DonatePartners.jsx). Shows how the data is consumed for the current Donate page.
- **Orcasound support flow component:** [`src/components/Donate/DonateOrcasound.tsx`](https://github.com/orcasound/orcahome/blob/main/src/components/Donate/DonateOrcasound.tsx). Open Collective and GitHub Sponsors links live inline here; flagged for follow-up data extraction.
- **Convention documentation:** [README "Donate page data" section](https://github.com/orcasound/orcahome/blob/main/README.md#donate-page-data).
- **Future Donate page tickets:** file via the [Donate page change issue template](https://github.com/orcasound/orcahome/issues/new?template=donate-page-change.yml) so the canonical paths and pre-merge verification checklist are pre-populated.

Refactor that established this convention: #264, PR #265.

**Comment by BT-Orcasound-Product** [comment]
Sprint 2 (5/18–5/29): Vicky Tran assigned to deploy version 2 of the redesigned donation page. Reassigned from prior assignee at 5/19 1 PM PT triage.

**Comment by BT-Orcasound-Product** [comment]
Correction to my earlier comment: "Vicky Tran" should be **Vicky Li** (GitHub: @fdsafewa). My error.
